src/lava/proc/io/sink.py

```diff
@@ -2,20 +2,22 @@
 # SPDX-License-Identifier: BSD-3-Clause
 # See: https://spdx.org/licenses/
 
-from typing import Tuple, Union
 import numpy as np
+from typing import Tuple, Union
 
-from lava.magma.core.process.process import AbstractProcess
-from lava.magma.core.process.ports.ports import InPort
 from lava.magma.core.process.variable import Var
-from lava.magma.core.sync.protocols.loihi_protocol import LoihiProtocol
-from lava.magma.core.model.py.ports import PyInPort
-from lava.magma.core.model.py.type import LavaPyType
+from lava.magma.core.process.process import AbstractProcess
+from lava.magma.core.process.ports.ports import InPort, RefPort
+
 from lava.magma.core.resources import CPU
 from lava.magma.core.decorator import implements, requires, tag
 from lava.magma.core.model.py.model import PyLoihiProcessModel
+from lava.magma.core.sync.protocols.loihi_protocol import LoihiProtocol
+from lava.magma.core.model.py.type import LavaPyType
+from lava.magma.core.model.py.ports import PyInPort, PyRefPort
 
 
+# Ring Buffer #################################################################
```

Comment:
This comment is probably not needed. Looks out of place with the many #

Reply:
sure will remove it